sklearn/cluster/_kmeans.py

```diff
@@ -72,7 +72,7 @@
 def kmeans_plusplus(
     X, n_clusters, *, x_squared_norms=None, random_state=None, n_local_trials=None
 ):
-    """Init n_clusters seeds according to k-means++.
+    """Init n_clusters seeds according to greedy k-means++ (default) or k-means++.
```

Comment:
Let's keep the headline simpler and only mention the distinction between variants in the docstring for `n_local_trials`.

```suggestion
    """Init n_clusters seeds according to k-means++.
```